crmd/join_client.c

```diff
@@ -30,6 +30,26 @@ void join_query_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, v
 
 extern ha_msg_input_t *copy_ha_msg_input(ha_msg_input_t * orig);
 
+/*!
```

Comment:
No, that's the most likely scenario (and what I used to reproduce). I would guess if a larger cluster were taken down to one node, it could happen there, too. It would be an atypical usage for sure; in the user's case that started this, it was a test script that exercised various scenarios.